packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -175,13 +177,13 @@ class SystemRulesTable extends React.Component {
         } else {
             // One rule was selected
             const index = ((page - 1) * itemsPerPage * 2) + Number(key);
-            if (currentRows[index].cells[columnIndices.REMEDIATIONS_COLUMN].original) {
+            if (tailoringEnabled || remediationsEnabled && currentRows[index].cells[columnIndices.REMEDIATIONS_COLUMN].original) {
```

Comment:
I'm a bit confused. `tailoringEnabled || remediationsEnabled` also happens down here as well: https://github.com/RedHatInsights/frontend-components/pull/392/files#diff-49f1e6a71fee60aaf0a07a8a1f8511aaR598

Reply:
You're right - onSelect won't even be called if those were not true. However, if I remove this check here, it would be something like 
`if (currentRows[index].cells[columnIndices.REMEDIATIONS_COLUMN].original)`
which would not allow me to select rules without remediations during tailoring.